packages/mobile/src/recipients/recipient.ts

```diff
@@ -13,10 +12,10 @@ import Logger from 'src/utils/Logger'
 const TAG = 'utils/recipient'
```

Comment:
Forgot this tag. Feels like the name and folder shouldn't be the same.

Reply:
Good catch!